update.sh

```diff
@@ -35,7 +35,61 @@ InstallWineRunner() {
   curl -sL "$url" | tar xfzv - -C "$dest_path"
   install_complete=true
   echo "Installation completed"
-  RestartLutrisCheck
+  Wannadelete
+}
+
+Wannadelete() {
```

Comment:
Lets rename this to `DeleteRestartPrompt()`, to keep it descriptive.

Reply:
you're right, that sounds better. It can be hard to come up with good function names ;)